api/PageTransitionEvent.json

```diff
@@ -0,0 +1,105 @@
+{
+  "api": {
+    "PageTransitionEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PageTransitionEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": null
```

Comment:
I just manually tested in Chromium 67, Firefox 61 and Opera 50 that these support both `PageTransitionEvent` and the `persisted` property.